src/fsharp/ast.fs

```diff
@@ -1742,7 +1755,7 @@ let mkSynDotBrackSeqSliceGet  m mDot arr (argslist:list<SynIndexerArg>) =
                        | SynIndexerArg.One x -> yield x
                        | _ -> () ]
     if notsliced.Length = argslist.Length then
-        SynExpr.DotIndexedGet(arr,[SynIndexerArg.One (SynExpr.Tuple(notsliced,[],unionRanges (Seq.head notsliced).Range (Seq.last notsliced).Range))],mDot,m)
```

Comment:
TBH the Microsoft internal build has always been a mystery to me :) Agreed we should align the builds as much as possible though, certainly the LKGs should be the same.

FWIW I suspect we don't need to build either a proto FSharp.Core nor a proto FSharp.Build with the LKG.  I didn't know the Microsoft internal build made a proto FSharp.Core and I don't know why it would need to - the LKG comes with the FSharp.Core it like to use.   There's no real guarantee the LKG will even be suitable for building the latest FSharp.Core.

As a historical note, the original duplication of ther project files crept in when someone (no one who is still here :)) duplicated all the project files way back in 2007 to make the DevDiv build projects, for reasons I never qutie understood. We continued using the original project files too. Misalignments crept in each time we needed to do an open source code drop for F# 2.0-3.1 - we needed a build that worked in the open and couldn't extract the DevDiv build files. So we put together something that worked based on the original project files. 

Anyway we've been living with two build systems on and off ever since that original duplication in 2007. It's amazing how a decision to duplicate things propagates down the years.  Now we're finally open source we can align things, let's do it.

Cheers
Don